.github/workflows/merge.yml

```diff
@@ -350,29 +350,32 @@ jobs:
           SN_LOG: "v"
         timeout-minutes: 2
 
-      - name: Estimate cost to create a vault
-        if: matrix.os != 'windows-latest'
+      - name: create local user data
         run: |
-          echo "test-file" > upload-test.txt
```

Comment:
Should we have `set -e` to error out here?

Reply:
set -e doesn't work for windows, let me separate them into two subjobs.

Reply:
Even if we set `shell: bash`?

Reply:
i tried using pswh shell. and added related macro there